tests/ttnn/unit_tests/operations/test_all_gather.py

```diff
@@ -368,7 +362,7 @@ def test_all_gather_on_t3000_post_commit(
 
 
 def run_line_all_gather(
-    all_devices,
+    pcie_device_mesh,
```

Comment:
We're losing coverage here.. `pcie_device_mesh` is a fixture that only returns the PCIE devices. You probably want to swap this with `t3k_device_mesh`

Reply:
Updated the code to use t3k_device_mesh